src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -2698,6 +2714,7 @@ def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_g
                      '--node-count 1 ' \
                      '--ssh-key-value={ssh_key_value} '
         create_cmd += f'--assign-identity {identity_id}' if user_assigned_identity else ''
```

Comment:
Need one trailing space. Otherwise, the case would fail. See [pipeline result](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=65523033&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=a26fe913-e2be-5062-b08e-d15f1acc7ea2&l=699).